security/user_checkers.rst

```diff
@@ -23,7 +23,7 @@ displayed to the user::
 
     namespace App\Security;
 
-    use App\Security\User as AppUser;
```

Comment:
Why not referred to `AppBundle` and `use AppBundle\Exception\AccountDeletedException;` was also lost?

Reply:
Sorry, I was on 3.4 branch.